packages/webapp/pages/squads/[handle]/edit.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { ReactElement } from 'react';
+import { NextSeo, NextSeoProps } from 'next-seo';
+import router, { useRouter } from 'next/router';
+import { useAuthContext } from '@dailydotdev/shared/src/contexts/AuthContext';
+import Unauthorized from '@dailydotdev/shared/src/components/errors/Unauthorized';
+import { SquadDetails } from '@dailydotdev/shared/src/components/squads/Details';
+import { SquadForm, editSquad } from '@dailydotdev/shared/src/graphql/squads';
+import { useBoot } from '@dailydotdev/shared/src/hooks/useBoot';
+import { useToastNotification } from '@dailydotdev/shared/src/hooks/useToastNotification';
+import {
+  ManageSquadPageContainer,
+  ManageSquadPageMain,
+  ManageSquadPageHeader,
+} from '@dailydotdev/shared/src/components/squads/utils';
+import { MangeSquadPageSkeleton } from '@dailydotdev/shared/src/components/squads/MangeSquadPageSkeleton';
+import { useQueryClient } from 'react-query';
+import { useSquad } from '@dailydotdev/shared/src/hooks';
+import {
+  GetStaticPathsResult,
+  GetStaticPropsContext,
+  GetStaticPropsResult,
+} from 'next';
+import { ParsedUrlQuery } from 'querystring';
+import {
+  Button,
+  ButtonSize,
+} from '@dailydotdev/shared/src/components/buttons/Button';
+import ArrowIcon from '@dailydotdev/shared/src/components/icons/Arrow';
+import { defaultOpenGraph, defaultSeo } from '../../../next-seo';
+import { getLayout as getMainLayout } from '../../../components/layouts/MainLayout';
+
+type EditSquadPageProps = { handle: string };
+
+const seo: NextSeoProps = {
+  title: 'Create your Squad',
+  openGraph: { ...defaultOpenGraph },
+  ...defaultSeo,
+};
+
+const EditSquad = ({ handle }: EditSquadPageProps): ReactElement => {
```

Comment:
Also generic note, don't forget we need tracking here probably.

Reply:
Discussion here https://dailydotdev.slack.com/archives/C0592NL7YCU/p1688983153989259

Reply:
We are going to use `page view`, no additional events needed.